### PR TITLE
Update to 1.0

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name = "Highlights"
+uuid = "eafb193a-b7ab-5a9e-9068-77385905fa72"
+version = "0.4"
```
Comment (mortenpi) [inline Project.toml]: How does registrator handle the missing patch version number? Should this be updated to be `0.4.0`?

Comment (KristofferC) [reply Project.toml]: Yes

Comment (KristofferC) [reply Project.toml]: Done

**Comment by mortenpi** [comment]
LGTM, thanks! CI failures due to #25.

**Pull request merged**